README.md

```diff
@@ -2,6 +2,12 @@
 
 Java SDK for Bandwidth's number management API.
 
+## Release Notes
+
+| Version | Notes |
+|:---|:---|
+| 0.6.0  | Fixed `quantity` and `tollFreeWildCardPattern` for search requests |
```

Comment:
How'd we get to `0.6.0`? I thought we had only released 0.1.0 previously.

Reply:
oof. You're right. I was looking at the messaging sdk version on accident

Reply:
Fixed 🎉